flickr_download/flick_download.py

```diff
@@ -169,6 +178,10 @@ def main():
                         default=None, help='Quality of the picture')
     parser.add_argument('-n', '--naming', type=str, metavar='NAMING_MODE',
                         default='title', help='Photo naming mode')
+    parser.add_argument('-f', '--folder_naming', type=str, metavar='FOLDER_NAMING',
+                        default='%Y/%m/%d', help='Photo folder naming')
+    parser.add_argument('-ft', '--file_times', action='store_true',
+                        default=False, help='Set file timestamp to photo taken timestamp')
```

Comment:
I'm happy with the feature, but you are changing the default to `False` here. For backwards compatibility you should use `True`

I would also suggest `Set file time to time when the photo was taken` as the help text